src/test/resources/rest-api-spec/test/mapper_attachments/00_basic.yaml

```
@@ -10,5 +10,5 @@
     - do:
         nodes.info: {}
 
-    - match:  { nodes.$master.plugins.0.name: mapper-attachments  }
+    - match:  { nodes.$master.plugins.0.name: elasticsearch-mapper-attachments  }
```

I think this is really trappy? We should just fail if this is not set.